src/localplayer.cpp

```diff
@@ -301,12 +300,10 @@ void LocalPlayer::move(f32 dtime, Environment *env, f32 pos_max_d,
 			v3f pf = intToFloat(p, BS);
 			v2f node_p2df(pf.X, pf.Z);
 			f32 distance_f = player_p2df.getDistanceFrom(node_p2df);
-			f32 max_axis_distance_f = MYMAX(
-					fabs(player_p2df.X-node_p2df.X),
-					fabs(player_p2df.Y-node_p2df.Y));
 
-			if(distance_f > min_distance_f ||
-					max_axis_distance_f > 0.5*BS + sneak_max + 0.1*BS)
+			if (distance_f > min_distance_f ||
+					fabs(player_p2df.X-node_p2df.X) > (.5+.1)*BS + sneak_max.X ||
+					fabs(player_p2df.Y-node_p2df.Y) > (.5+.1)*BS + sneak_max.Z)
```

Comment:
For 2 lines above:
Spaces around - + *
Also, .5 + .1 can be replaced with .6 plus possibly a comment (if necessary) that this is .5 + .1